static/bidder-info/connektai.yaml

```diff
@@ -0,0 +1,20 @@
+aliasOf: "xeworks"
+endpoint: "http://rtb.connektai.live/?pid={{.SourceId}}&host={{.Host}}&s=pbs"
+maintainer:
+  email: "adops@connekt.ai"
```

Comment:
We've sent an email to this address. Please respond with a message of 'received' to confirm it was received.

Reply:
Confirmed.